datahub-web-react/src/app/ingest/source/builder/RecipeBuilder.tsx

```diff
@@ -67,14 +80,18 @@ function RecipeBuilder(props: Props) {
 
     return (
         <div>
-            <ButtonsWrapper>
-                <StyledButton type="text" isSelected={isViewingForm} onClick={() => switchViews(true)}>
-                    <FormOutlined /> Form
-                </StyledButton>
-                <StyledButton type="text" isSelected={!isViewingForm} onClick={() => switchViews(false)}>
-                    <CodeOutlined /> YAML
-                </StyledButton>
-            </ButtonsWrapper>
+            {(type === LOOKER || type === LOOK_ML) && <LookerWarning type={type} />}
+            <HeaderContainer>
+                <Title level={5}>{sourceConfigs?.displayName} Recipe</Title>
```

Comment:
can you add `style={{ marginBottom: 0 }}` as an inline style to `Title` here? For some reason ant design always adds margin to the bottom of `Typography.Title` and you can't remove it using styled components so you have to do it inline.

Reply:
Checked and fixed!!